.ci/Jenkinsfile

```diff
@@ -350,7 +353,7 @@ stage('Build') {
                 sh "mkdir -p $buildOutputFolder"
 
                 archiveArtifacts(artifacts: artifactsGlob, fingerprint: true, allowEmptyArchive: true)
-                junit(allowEmptyResults: true, testResults: junitGlob)
+                junit(allowEmptyResults: true, testResults: junitGlob, checksName: 'Tests')
```

Comment:
How will this work if there are multiple failures across different containers?  Not sure if this is really an issue given we do early termination?

Reply:
The Junit results are merged and collected from all spawned pods. So, multiple failures will show up here.